examples/run_examples.py

```diff
@@ -62,18 +62,25 @@
 try:
     from daal4py.oneapi import sycl_context
     sycl_extention_available = True
-except:
+except ModuleNotFoundError:
     sycl_extention_available = False
 
 if sycl_extention_available:
     try:
         with sycl_context('gpu'):
-            gpu_available = True
             availabe_devices.append("gpu")
-    except:
-        gpu_available = False
-    availabe_devices.append("host")
-    availabe_devices.append("cpu")
+    except RuntimeError:
+        pass
+    try:
+        with sycl_context('host'):
+            availabe_devices.append("host")
```

Comment:
This means nothing since `req_device` contains only one requirement for one example to run only on gpu.
The suggestion is to add [here](https://github.com/IntelPython/daal4py/blob/f4b79297a4661d4a32e7cbe87933bafbdd5012b4/examples/run_examples.py#L132) generating requirements on host and cpu devices for all sycl examples. In that case, you would not need to change sklearn example at all.